### ISSUE TITLE
align.openParenCallSite: inconsistent behaviour

### ISSUE THREAD

**Issue by fedragon** [scalameta/scalafmt#1051]
Hello, 
I see some inconsistent behavior with the `align.openParenCallSite`... at least, based on my expectations :)

Here are the details:

- **Version**: 1.2.0
- **Integration**: none (CLI)
- **Configuration**:
```
align = none
align.openParenCallSite = false
newlines.penalizeSingleSelectMultiArgList = false
```

## Steps

I gave `scalafmt` the following input

```bash
class MyClass {
  private[mypackage] def function1(aaaaaaaaaaaa: String, bbbbbbbbbbb: String, cccccccccccc: Option[String]): DBIO[Option[MyResultTypeName]] = ???

  private[mypackage] def function2(aaaaaaaaaaaa: String, bbbbbbbbbbb: String, cccccccccccc: Option[String]) = ???
}
```

## Problem

Scalafmt formats `function1` as follows:
```scala

  private[mypackage] def function1(
      aaaaaaaaaaaa: String,
      bbbbbbbbbbb: String,
      cccccccccccc: Option[String]): DBIO[Option[MyResultTypeName]] = ???
```

and `function2` as follows:
```scala

  private[mypackage] def function2(aaaaaaaaaaaa: String,
                                   bbbbbbbbbbb: String,
                                   cccccccccccc: Option[String]) = ???
```

(as if I had set `align.openParenCallSite = true`)

## Expectation

I would like both functions to look like `function1`, meaning:
```scala
  private[mypackage] def function1(
      aaaaaaaaaaaa: String,
      bbbbbbbbbbb: String,
      cccccccccccc: Option[String]): DBIO[Option[MyResultTypeName]] = ???
```

In a nutshell, I would like arguments to be indented as specified by `align.openParenCallSite` whenever the definition doesn't fit in `maxColumn`.

## Workaround

I've found that by explicitly setting a very long return type in `function2`, the behaviour becomes consistent, but then again, it really depends on the type name length.

**Comment by olafurpg** [comment]
Have you tried `align.openParenDefnSite = false`?

**Comment by fedragon** [comment]
Oh, I wasn't aware of that configuration option (and I now realize that expecting something from `openParenCallSite` as wrong to begin with). That works indeed, thanks a lot @olafurpg, you can disregard this issue!
